src/redux/rockets/actions.js

```diff
@@ -0,0 +1,18 @@
+export const ADD_RESERVATION = 'ADD_RESERVATION';
+export const DELETE_RESERVATION = 'DELETE_RESERVATION';
+export const GET_ROCKETS = 'GET_ROCKETS';
```

Comment:
Don't you think it's better to use the Ducks pattern for our actions? it's among the best practices.